data/scenarios/Testing/201-require/00-ORDER.txt

```diff
@@ -1,8 +1,8 @@
 201-require-device.yaml
 201-require-device-creative.yaml
 201-require-device-creative1.yaml
-201-require-entities.yaml
-201-require-entities-def.yaml
+201-stock-entities.yaml
+201-stock-entities-def.yaml
```

Comment:
It's possible that this should now be moved out from this set of tests/scenarios?